docs/en_US/install/wiiu.md

```diff
@@ -55,17 +55,34 @@ Bleeding edge builds have not been widely tested, and do not guarantee any stabi
 
 <img src="/assets/images/docs/install/wiiu/aroma/inkay-nightly-link.png" width="100%">
 
+## Nimble
+Nimble is a Aroma plugin that redirects the PolicyList to Pretendo. Download the latest version from [Nimble's latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the `30_nimble.rpx` file from the GitHub page.
+
+<img src="/assets/images/docs/install/wiiu/aroma/nimbus-install.png" width="100%">
+
 ## Installation
 Place the downloaded `Inkay-pretendo.wps` file on your SD card at `sd:/wiiu/environments/aroma/plugins`. It's normal for other Aroma plugins to be in this folder too.
 
 <img src="/assets/images/docs/install/wiiu/aroma/sd-card.png" width="100%">
 
+Place the file `30_nimble.rpx` on your SD card inside the folder called `sd:/wiiu/environments/aroma/modules/setup`. 
+
+<img src="/assets/images/docs/install/wiiu/pnid/nimbus-sd.png" width="100%">
+
+Turn on your console.
+
 Place your SD card back into your console and boot like normal. You should see a notification of your connection:
 
 <img src="/assets/images/docs/install/wiiu/aroma/working-notification.jpg" width="100%">
 
 You are now connected to Pretendo Network.
 
+Press the button combo `L + Down + SELECT` on the Wii U GamePad to open the Aroma plugin menu.
+
+<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">
+
+Choose the plugin 'Inkay', then click A on 'BOSS settings' inside of Inkay, then click A on 'Unregister Wara Wara Plaza BOSS tasks' after that restart your console fully.
```

Comment:
I don't blame you for not knowing this, but the bleeding-edge version of Inkay has actually changed these texts to be less technical and confusing.
It now says "Other settings" and "Reset Wara Wara Plaza". That's only in bleeding-edge though, not latest - @jonbarrow thoughts?

https://github.com/PretendoNetwork/Inkay/commit/216942cb64400a2140cbd8ed9c0bee2e8e7a2495#diff-6b2f0a449fdefd8930e23ef0dcd752beec69242e1303d77653f047c5e0766385R133

Reply:
Should I make it temporarily like 'Reset Wara Wara Plaza' OR 'Unregister Wara Wara Plaza BOSS tasks', etc?

Reply:
> Should I make it temporarily like 'Reset Wara Wara Plaza' OR 'Unregister Wara Wara Plaza BOSS tasks', etc?

Ah crap, on my alt, whoops!

Reply:
Ehhh, I guess leave it - we only just pushed Inkay v2.4 out so it'll be a bit before we do another release (unless something comes up)

Reply:
> Ehhh, I guess leave it - we only just pushed Inkay v2.4 out so it'll be a bit before we do another release (unless something comes up)

Just saw this tight after i pushed out the commit lol, will revert it

Reply:
Reverted.